tests_config/run_spec.js

```diff
@@ -47,19 +47,18 @@ const unstableTests = new Map(
   })
 );
 
-// This is temporary.
-// This will be removed once https://github.com/babel/babel/issues/12915 is fixed
 const unstableAstTests = new Map(
   [
     [
-      "flow/function-parentheses/single.js",
-      (options) =>
-        options.parser.startsWith("babel") && options.arrowParens !== "avoid",
+      "json/json/propertyKey.json",
+      (options) => options.parser === "json-stringify",
     ],
     [
-      "flow/function-parentheses/test.js",
-      (options) => options.parser.startsWith("babel"),
+      "json/json/positive-number.json",
+      (options) => options.parser === "json-stringify",
     ],
+    ["json/json/json5.json", (options) => options.parser === "json-stringify"],
+    ["json/json/array.json", (options) => options.parser === "json-stringify"],
```

Comment:
In now `parserExpression` returns AST includes `tokens` so may break AST idempotency for `json-stringify` parser.

Reply:
I see `tokens` in `ignoredProperties` in `clean.js`. Why does that property still affect idempotence tests?

Reply:
Probably `ignoreProperties` are ignored by `messageAST`, are not by idempotence tests.

https://github.com/prettier/prettier/blob/main/src/main/massage-ast.js#L13-L18

Reply:
It should work, I'll take a look.

Reply:
`tokens` was not ignored in `estree-json` printer.